<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FocusUser</class>
 <widget class="QDialog" name="FocusUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1193</width>
    <height>601</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>查看你都关注了谁</string>
  </property>
  <property name="windowIcon">
   <iconset resource="img.qrc">
    <normaloff>:/getheadimg.jpg</normaloff>:/getheadimg.jpg</iconset>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>391</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>389</width>
      <height>459</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView_4">
       <property name="font">
        <font>
         <family>.萍方-简</family>
         <pointsize>11</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 200), stop:0.09 rgba(246, 189, 237, 120), stop:0.14 rgba(194, 207, 246, 120), stop:0.19 rgba(184, 160, 168, 60), stop:0.25 rgba(171, 186, 248, 120), stop:0.32 rgba(243, 248, 224, 120), stop:0.385 rgba(249, 162, 183, 120), stop:0.47 rgba(255, 139, 124, 80), stop:0.58 rgba(251, 205, 202, 120), stop:0.65 rgba(170, 128, 185, 60), stop:0.75 rgba(252, 222, 204, 120), stop:0.805 rgba(206, 122, 218, 120), stop:0.86 rgba(254, 223, 175, 120), stop:0.91 rgba(254, 236, 244, 120), stop:1 rgba(255, 191, 221, 200));</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(244, 185, 241);border-radius:5px</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QLable{
backgroundcolor:rgb(244, 185, 241);
color:rgb(255, 170, 255);
	font: 12pt &quot;新蒂黑板报底字&quot;;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string>Your Attention</string>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>回答列表</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>731</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>729</width>
      <height>179</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QTableView" name="tableView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>.萍方-简</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 200), stop:0.09 rgba(246, 189, 237, 120), stop:0.14 rgba(194, 207, 246, 120), stop:0.19 rgba(184, 160, 168, 60), stop:0.25 rgba(171, 186, 248, 120), stop:0.32 rgba(243, 248, 224, 120), stop:0.385 rgba(249, 162, 183, 120), stop:0.47 rgba(255, 115, 124, 100), stop:0.58 rgba(251, 205, 202, 120), stop:0.65 rgba(170, 128, 185, 60), stop:0.75 rgba(252, 222, 204, 120), stop:0.805 rgba(206, 122, 218, 120), stop:0.86 rgba(254, 223, 175, 120), stop:0.91 rgba(254, 236, 244, 120), stop:1 rgba(255, 191, 221, 200));</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>560</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>尾页</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>260</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>尾页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>320</y>
     <width>731</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>729</width>
      <height>219</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <family>.萍方-简</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(245, 224, 176, 200), stop:0.09 rgba(246, 189, 237, 120), stop:0.14 rgba(194, 207, 246, 120), stop:0.19 rgba(184, 160, 168, 60), stop:0.25 rgba(171, 186, 248, 120), stop:0.32 rgba(243, 248, 224, 120), stop:0.385 rgba(249, 162, 183, 120), stop:0.47 rgba(255, 115, 124, 100), stop:0.58 rgba(251, 205, 202, 120), stop:0.65 rgba(170, 128, 185, 60), stop:0.75 rgba(252, 222, 204, 120), stop:0.805 rgba(206, 122, 218, 120), stop:0.86 rgba(254, 223, 175, 120), stop:0.91 rgba(254, 236, 244, 120), stop:1 rgba(255, 191, 221, 200));</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>30</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>问题列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_10">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>550</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>新蒂黑板报底字</family>
    </font>
   </property>
   <property name="text">
    <string>首页</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="img.qrc"/>
 </resources>
 <connections/>
</ui>
